src/components/v5/common/CompletedAction/partials/PaymentBuilder/partials/EditStep/partials/EditRequests/hooks.ts

```diff
@@ -0,0 +1,49 @@
+import { type ExpenditureSlot } from '~types/graphql.ts';
+
+const useCountChanges = (
+  newSlots: ExpenditureSlot[],
+  oldSlots: ExpenditureSlot[],
+) => {
+  let changes = 0;
+
+  if (newSlots.length !== oldSlots.length) {
+    changes += newSlots.length - oldSlots.length;
+  }
```

Comment:
This looks like it would subtract if there were less new slots than old ones, even though normally it shouldn't happen. Still, it feels sensible to wrap this in `Math.abs`